pkg/logger/logger.go

```diff
@@ -0,0 +1,55 @@
+package logger
+
+import (
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+var logLevel logrus.Level
+
+var stdLoggerConfigd = false
+
+func ConfigLogger(log *logrus.Logger, level string) error {
```

Comment:
Is this necessarily? Like I think just the splunk hook should be here, configuration can/should be handled separately.